cassandra-config.yml.sample

```diff
@@ -9,6 +9,13 @@ instances:
           username: testUser
           password: testPassword
 
+          # Keystore and Truststore settings are only necessary when JMX
+          # has been configured to use TLS encryption.
+          key_store: /path/to/keystore.p12
+          key_store_password: keystorePassword
+          trust_store: /path/to/truststore.p12
+          trust_store_password: truststorePassword
+
```

Comment:
```suggestion
```
I see that only required parameters are listed in this config sample , so I think we can remove this and add the parameters in the [docs](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/cassandra-monitoring-integration) , i can add that.

Reply:
Good point. I'll remove them.